diktat-rules/src/main/kotlin/com/saveourtool/diktat/ruleset/rules/chapter3/files/IndentationRule.kt

```diff
@@ -64,7 +64,7 @@ import org.jetbrains.kotlin.KtNodeTypes.VALUE_ARGUMENT
 import org.jetbrains.kotlin.KtNodeTypes.VALUE_ARGUMENT_LIST
 import org.jetbrains.kotlin.KtNodeTypes.VALUE_PARAMETER_LIST
 import org.jetbrains.kotlin.lexer.KtTokens.WHITE_SPACE
-import org.cqfn.diktat.ruleset.utils.isWhiteSpaceWithNewline
+import com.saveourtool.diktat.ruleset.utils.isWhiteSpaceWithNewline
```

Comment:
invalid ordering in imports. It's strange that diktat doesn't detect it

Reply:
Why invalid? Local imports should always go after non-local company imports

Reply:
https://diktat.saveourtool.com/info/guide/diktat-coding-convention.html#r3.1.3

Reply:
According to this:
```
2. Imports of packages used internally in your organization
3. Imports from other non-core dependencies
```

I guess, line 67 should be moved to block from lines 9:38